artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -84,25 +82,21 @@ global:
 
 .Procedures
 
-. On the {product-very-short} platform, navigate to the *Catalog* and select an appropriate component.
+. Select the *Catalog* tab and choose the component that you want to use.
 
-. Select *CD* tab to view insights into deployments managed by Argo CD. 
+. Select the *CD* tab to view insights into deployments managed by Argo CD. 
 
 +
 image::rhdh-plugins-reference/argocd.png[CD tab Argo CD]
 
-. For deployment details:
-
-+
-* Select an appropriate card to view the deployment details (for example, commit message, author name, and deployment history) in a sidebar.
+. Select an appropriate card to view the deployment details (for example, commit message, author name, and deployment history).
 
 +
 image::rhdh-plugins-reference/sidebar.png[Sidebar]
 
-+ 
-* Click the link icon to open the deployment details in Argo CD.
+.. Click the link icon (image:rhdh-plugins-reference/link.svg[width=13px, height=13px, align=center]) to open the deployment details in Argo CD.
 
-. Select the *Overview* tab and navigate to the Deployment summary section to review the summary of your application's deployment across namespaces. Additionally, select an appropriate Argo CD app to open the deployment details in Argo CD, or select a commit from the Revision column to review the changes in GitLab or GitHub.
+. Select the *Overview* tab and navigate to the Deployment summary section to review the summary of your application's deployment across namespaces. Additionally, select an appropriate Argo CD app to open the deployment details in Argo CD, or select a commit id from the Revision column to review the changes in GitLab or GitHub.
```

Comment:
as per IBMSG, we should be using the abbreviation for identifier, 
![image](https://github.com/user-attachments/assets/94e77cac-db4a-4140-a1c5-709d61b7ef32)
```suggestion
. Select the *Overview* tab and navigate to the Deployment summary section to review the summary of your application's deployment across namespaces. Additionally, select an appropriate Argo CD app to open the deployment details in Argo CD, or select a commit ID from the Revision column to review the changes in GitLab or GitHub.
```